apps/settings/js/bluetooth.js

```diff
@@ -664,13 +666,14 @@ navigator.mozL10n.once(function bluetoothSettings() {
         }
       } else {
         // show pair process fail.
-        var msg = _('error-pair-title');
+        var l10nId = 'error-pair-title';
         if (errorMessage === 'Repeated Attempts') {
-          msg = msg + '\n' + _('error-pair-toofast');
+          l10nId = 'error-pair-toofast';
         } else if (errorMessage === 'Authentication Failed') {
-          msg = msg + '\n' + _('error-pair-pincode');
+          l10nId = 'error-pair-pincode';
```

Comment:
Oh! You remind me that here is a legacy code for Bluetooth API version 1. Sorry I didn't think of it. I asked Ian who wrote the newer version (bluetooth/panel.js#L593-L599) and he said UX reviewed that part. So feel free to ignore my previous comments.